src/lib.rs

```diff
@@ -3817,7 +3934,7 @@ impl TransportParams {
 
         // TODO: encode preferred_address
 
-        if tp.active_conn_id_limit != 0 {
+        if tp.active_conn_id_limit != 2 {
```

Comment:
Does this need to be `less than 2`?

Reply:
No, this is to decide whether to encode the TP or not (i.e. if the value is the default -> don't encode as it is implicit), and the default value is "2".